pages/invites/index.js

```diff
@@ -51,9 +51,7 @@ function InviteForm () {
             gift: Number(gift), limit: limit ? Number(limit) : limit
           }
         })
-        if (error) {
-          throw new Error({ message: error.String() })
-        }
+        if (error) throw error
```

Comment:
I'm not sure why I ever did this weird error wrapping.

Reply:
Mhh, the only thing that I can tell this does it to lose where the error actually happened, lol.